.changeset/swift-panthers-play.md

```diff
@@ -0,0 +1,5 @@
+---
+'@accounts/mongo-sessions': minor
+---
+
+Changed idProvider to idSessionProvider in mongo-sessions
```

Comment:
As this is a breaking change can we provide more info for the user to explain how to upgrade?

Reply:
That's an excellent point. This is addressed in b89b6cdf77c759f9dc2dc7de936becf805184259